pkgs/servers/isso/default.nix

```diff
@@ -63,5 +63,7 @@ with python3Packages; buildPythonApplication rec {
     homepage = "https://posativ.org/isso/";
     license = licenses.mit;
     maintainers = with maintainers; [ fgaz ];
+    # never built on aarch64-darwin since first introduction in nixpkgs
```

Comment:
Remove, eval fail? I think stdenv is not in this scope, but I guess it is a bit weird with nodePackages

Reply:
No, this is not an eval fail. I just missed that stdenv was inherited and was distracted by isso being marked as 'changed' by ofborg.